app/controllers/push_controller.rb

```diff
@@ -29,9 +29,13 @@ def callback
     video.published_at = entry['published']
     video.title = entry['title']
 
-    if video.published_at < 1.hour.ago
-      # ignore, just someone updating an ancient video
-    elsif video.new_record?
+    if video.new_record? && video.published_at < 1.day.ago
```

Comment:
don't think threshold really had to increase, but may as well if we can guarantee `video`s live for at least a day